docs/how-to/index.md

```diff
@@ -0,0 +1,7 @@
+# How To
+
+Welcome to the How To section.
+
+Here you will find How To articles which cover various topics at a higher level than the Guide or Reference.
```

Comment:
Perhaps link to the guide and reference here?

Reply:
The reader can bloody well just look up at the menu bar.

Reply:
😆